### PR TITLE
Add FISTA and CD julia solvers

### PR DESCRIPTION
**Pull request by TheoGuyard**
This PR adds two new solvers : Julia-CD and Julia-FISTA

### CODE REVIEW

File: solvers/julia_fista.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
```
Comment (mathurinm) [inline solvers/julia_fista.py:1]: @TheoGuyard could you make this an option of `julia_pgd` to reduce code duplication and mimick what's done in python_pgd?

File: solvers/julia_fista.jl [hunk 2]
```diff
@@ -0,0 +1,34 @@
+using Core
+using LinearAlgebra
+
+st(w::Vector{Float64}, t::Float64) = @. sign(w) * max(abs(w) - t, 0.)
+
+function solve_lasso(
```
Comment (mathurinm) [inline solvers/julia_fista.jl:6]: could this be merged with the Julia ISTA solver?

File: solvers/julia_fista.py (hunk 1 above)
Comment (TheoGuyard) [reply solvers/julia_fista.py:1]: Yep, I'm on it 👍

File: solvers/julia_cd.jl [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            w[j] = st(wj + (Xj' * r) / L[j], lmbd / L[j])
+            diff = wj - w[j]
+            if diff != 0.
+                r += diff * Xj
+            end
+        end
+    end
+
+    return w
+end
```
[the remark this answers is not in the document]
Comment (TheoGuyard) [reply solvers/julia_cd.jl:29]: Crap, I still need to configure my black linter...

**Comment by TheoGuyard** [comment]
Everything should be ok now

**Pull request closed without merging**